app/routes/admin.js

```diff
@@ -98,27 +95,27 @@ var removeUserRole = function (username, next) {
 }
 
 var addUpdateUserRole = function (username, rights, loggedInUsername) {
-  var loggedInUser
-  var user
-  var role
   return userRoleService.getUser(loggedInUsername).then(function (result) {
-    loggedInUser = result
+    var loggedInUser = result
     return userRoleService.getUser(username).then(function (result) {
-      user = result
-      return userRoleService.getRole(rights).then(function (result) {
-        role = result
+      var user = result
```

Comment:
Could also remove all these `var user/loggedInUser = result` by doing `.then(function (user/loggedInUser) {`

But happy with this